app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for individ in people:
```

Comment:
Try to rewrite in one line. Hint: list comprehention 🤫

Reply:
Could you please a bit clarify because I don`t understand. Do I need to rewrite the whole for loop in one line?

Reply:
Why do u need to create a blank list `person_list` and then make a separate for loop for creating an instances of People class
U can use list comprehension here for list + instances creating )